src/documentation/pages/components/button/examples/_danger-lumen-centurylink.vue

```diff
@@ -0,0 +1,48 @@
+<template lang="pug">
+  <ComponentExample titleSize="h2" title="Danger" id="danger-lumen-centurylink" :tabs="exampleTabs">
```

Comment:
we don't have "h2" as a possible value for titleSize. You'd need to add it to ComponentExample first, but i don't think h2 is the right header for example titles. It should be the default h3 (or h4) 🤔

Reply:
I have compared the prod and i could see h2 tags in the DOM. h tags are a bit different in chi vue when compared to original doc. But i can update to match the UI.